docker/Dockerfile-bmltdebug

```diff
@@ -1,10 +1,14 @@
 FROM public.ecr.aws/m3g5z4b2/bmlt-root-server:latest
 
-ENV PHP_INI_PATH "/usr/local/etc/php/php.ini"
+ENV PHP_INI_PATH /etc/php/7.2/apache2/php.ini
+ENV PHP_XDEBUG_ENABLED: 1
 
-RUN pecl install xdebug-2.7.2 \
-    && /usr/local/bin/docker-php-ext-enable xdebug \
-    && echo "zend_extension=$(find /usr/local/lib/php/extensions/ -name xdebug.so)" >> ${PHP_INI_PATH} \
+RUN apt-get update \
+    && apt-get -yqq install \
```

Comment:
I like to put my &&s at the end of the previous lines instead 🤷‍♂️

Reply:
yea it does look cleaner that way